app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -251,6 +271,31 @@ export const SendBitcoinScreen: ScreenType = ({
       userDefaultWalletIdQuery({ variables: { username: destination } })
       toastShow(translate("SendBitcoinScreen.usernameNotFound"))
       return
+    } else if (paymentType == "lnurl") {
+      const satAmount = primaryAmount.value * 1000
+      fetch(`${lnurlCb}?amount=${satAmount}&comment=${memo}`)
+        .then(res => res.json())
+        .then(res => {
+          if (res.status && res.status === "ERROR") {
+            console.log('invoice fetch error ', res.reason)
+          } else {
```

Comment:
We should probably extract this into its own function

Reply:
done